# config-top.h should be included by default in config.h
# otherwise some of the macros aren't visible.
# Solaris-specific patch. Not for upstream.
--- config.h.in	2009-08-18 20:34:36.000000000 -0700
+++ config.h.in	2010-09-01 10:21:00.756734104 -0700
@@ -21,6 +21,8 @@
 #ifndef _CONFIG_H_
 #define _CONFIG_H_
 
+#include "config-top.h"
+
 /* Configuration feature settings controllable by autoconf. */
 
 /* Define JOB_CONTROL if your operating system supports
